docs/next/guides/getting-started/installation.md

```diff
@@ -41,41 +86,86 @@ Run the following command in your terminal
   ```
 
   </code-block>
-  <code-block title="Nuget">
+  <code-block title="NuGet">
 
   ```bash
   PM> Install-Package Handsontable
   ```
 
   </code-block>
-  <code-block title="CDN">
+</code-group>
 
-  ```html
-  // minified JS
-  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js"></script>
+### Using a CDN
 
-  // minified CSS
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.css" />
-  ```
+To get Handsontable's files from a CDN, use the following locations:
 
-  </code-block>
-</code-group>
+```js
+https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js
 
-## Create a placeholder
+https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.css
+```
+
+## Import Handsontable's JavaScript
+
+Import Handsontable's JavaScript into your application.
+
+### Full distribution
+
+#### Using CommonJS or a package manager
+
+If you're using Handsontable as a CommonJS package, or as an ECMAScript module (using a package manager), import the full distribution of Handsontable as a JavaScript file.
+
+Use your bundler's preferred importing method. For example:
+
+```js
+import Handsontable from 'handsontable';
+```
+
+#### Using UMD
```

Comment:
Sadly most people who use UMD, don't know that it is called UMD. Let's change the text of the heading.

```suggestion
#### Using the `script` tag
```